app/models/book.rb

```diff
@@ -35,6 +36,6 @@ def attributes
       twitter_card_image_url: nil, trailer_video_url: nil, version_description: nil,
       professional: nil, difficulty: nil, platform: nil, language: nil, editor: nil, domains: [],
       categories: [], who_is_this_for: nil, covered_concepts: nil, hide_chapter_numbers: nil,
-      in_flux: nil }.stringify_keys
+      in_flux: nil, forum_url: nil, pages: nil, short_description: nil, recommended_skus: nil }.stringify_keys
```

Comment:
Is it possible for `recommended_skus`  to be parsed and sent as an array of strings? The data type in alexandria is an array type. If not, that's ok, I can do the parsing in alexandria.

Reply:
Oh, yes it should be. tbh, although I put `nil` or `[]` here, I'm yet to understand that it makes any difference. I thought it was the default value? but that doesn't seem to be the case. I'll update it.